### ISSUE TITLE
REFER transfer support

### ISSUE THREAD

**Issue by vhatzist** [gmaruzz/saraphone#29]
Does SaraPhone have REFER transfer support? If not, then is this planned for the future?

**Comment by gmaruzz** [comment]
nope, it does not support REFER (eg, the library 0.7.8 does not support it)
yes, planned for future
